pkg/types/types.go

```diff
@@ -29,18 +29,18 @@ func Init(nodeName string) {
 
 type CommonData struct {
 	// Node where the event comes from
-	Node string `json:"node,omitempty"`
+	Node string `json:"node,omitempty" column:"node,width:16" columnTags:"kubernetes"`
 
 	// Pod namespace where the event comes from, or empty for host-level
 	// event
-	Namespace string `json:"namespace,omitempty"`
+	Namespace string `json:"namespace,omitempty" column:"namespace,width:16" columnTags:"kubernetes"`
 
 	// Pod where the event comes from, or empty for host-level event
-	Pod string `json:"pod,omitempty"`
+	Pod string `json:"pod,omitempty" column:"pod,width:30" columnTags:"kubernetes"`
```

Comment:
Is this fine to have a width for the Pod almost twice than we have for namespace and node?

Reply:
As per the discussion about container width, if we use 25 for containers, if they belong to a pod that is part of a daemon set, then they will be for sure longer than 25. Do you think it happens very few times?

Reply:
BTW, I added the `ellipsis:left`.

Reply:
I think pod is a good candidate for `ellipsis:middle` - the start is usually the informative part (what is it) and the end is has got the unique identifier.

Regarding 25/30 characters, what we should keep in mind: when autoscaling is enabled, the columns will be extended by their ratios (if a column takes 1/5 of the size, it will do so after scaling as well).

Reply:
I think nodes and namespaces can be very long as well. What about using the same width for node, namespace, pod and container?

Reply:
That's true. AKS nodes: `aks-agentpool-31885719-vmss00000X`. 

Yeah, we could have 30 for all of them, but enable autoscaling by default so that it also works well on small screens. Then, for the ellipsis, we could start using `ellipsis:middle` for the pod and namespace, and `ellipsis:end` for the node and container name. WDYT?

Reply:
Hmm, I think `ellipsis:middle` would also make sense for the node (because those often start or end with the unique part). I don't know if it's common for namespaces to have some kind of unique identifier as well - if not, I would suggest using `ellipsis:end`.

Reply:
Maybe @alban can make some suggestions about this topic

Reply:
To conclude:

- Width: We agreed to use the same for all of them: `30`. And enable autoscale so that it also works well on small screens.
- Ellipsis: 
  - Node and Pod: `middle`.
  - Namespace and Container: `end` (Default).

Do we all agree?